src/transformers/integrations/bitsandbytes.py

```diff
@@ -39,6 +41,8 @@ class `Int8Params` from `bitsandbytes`.
             The value of the tensor (useful when going from the meta device to any other device).
         fp16_statistics (`torch.HalfTensor`, *optional*):
             The list of fp16 statistics to set on the module, used for serialization.
```

Comment:
```suggestion
            The list of fp16 statistics to set on the module, used for 8-bit serialization.
```

Reply:
done with combinig `fp16_statistics` and `quantized_stats`